### ISSUE TITLE
Build error in ts.h

### ISSUE THREAD

**Issue by Saentist** [braice/MuMuDVB#295]
```make
# make
Making all in src
make[1]: влизане в директория „/opt/MuMuDVB/src“
make  all-am
make[2]: влизане в директория „/opt/MuMuDVB/src“
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf.o -MD -MP -MF .deps/autoconf.Tpo -c -o autoconf.o autoconf.c
mv -f .deps/autoconf.Tpo .deps/autoconf.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT crc32.o -MD -MP -MF .deps/crc32.Tpo -c -o crc32.o crc32.c
mv -f .deps/crc32.Tpo .deps/crc32.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT log.o -MD -MP -MF .deps/log.Tpo -c -o log.o log.c
In file included from mumudvb.h:36,
                 from log.c:58:
ts.h:194:3: error: unknown type name ‘u_char’
  194 |   u_char adaptation_field_length  :8;
      |   ^~~~~~
ts.h:205:3: error: unknown type name ‘u_char’
  205 |   u_char discontinuity_indicator  :1;
      |   ^~~~~~
ts.h:206:3: error: unknown type name ‘u_char’
  206 |   u_char random_access_indicator  :1;
      |   ^~~~~~
ts.h:207:3: error: unknown type name ‘u_char’
  207 |   u_char elementary_stream_priority_indicator :1;
      |   ^~~~~~
ts.h:208:3: error: unknown type name ‘u_char’
  208 |   u_char PCR_flag    :1;
      |   ^~~~~~
ts.h:209:3: error: unknown type name ‘u_char’
  209 |   u_char OPCR_flag    :1;
      |   ^~~~~~
ts.h:210:3: error: unknown type name ‘u_char’
  210 |   u_char splicing_point_flag   :1;
      |   ^~~~~~
ts.h:211:3: error: unknown type name ‘u_char’
  211 |   u_char transport_private_data_flag  :1;
      |   ^~~~~~
ts.h:212:3: error: unknown type name ‘u_char’
  212 |   u_char adaptation_field_extension_flag :1;
      |   ^~~~~~
ts.h:217:3: error: unknown type name ‘u_char’
  217 |   u_char transport_private_data_length  :8;
      |   ^~~~~~
make[2]: *** [Makefile:493: log.o] Error 1
make[2]: излизане от директория „/opt/MuMuDVB/src“
make[1]: *** [Makefile:346: all] Error 2
make[1]: излизане от директория „/opt/MuMuDVB/src“
make: *** [Makefile:347: all-recursive] Error 1
```
Over internet there is sugestion about changing **u_char** to **uint8_t**.

OS info
````bash
# uname -a && lsb_release -a
Linux ET1610PT 5.13.0-40-generic #45~20.04.1-Ubuntu SMP Mon Apr 4 09:38:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.4 LTS
Release:        20.04
Codename:       focal```

**Comment by anphsw** [comment]
It seems like it fixed in #296

**Comment by trollcop** [comment]
Yes, indeed, fixed.

**Comment by Saentist** [comment]
This is fixed but now have other error's with not allow compilation to finist

```make
#make
Making all in src
make[1]: влизане в директория „/opt/MuMuDVB/src“
make  all-am
make[2]: влизане в директория „/opt/MuMuDVB/src“
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf.o -MD -MP -MF .deps/autoconf.Tpo -c -o autoconf.o autoconf.c
mv -f .deps/autoconf.Tpo .deps/autoconf.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT crc32.o -MD -MP -MF .deps/crc32.Tpo -c -o crc32.o crc32.c
mv -f .deps/crc32.Tpo .deps/crc32.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT log.o -MD -MP -MF .deps/log.Tpo -c -o log.o log.c
mv -f .deps/log.Tpo .deps/log.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT multicast.o -MD -MP -MF .deps/multicast.Tpo -c -o multicast.o multicast.c
mv -f .deps/multicast.Tpo .deps/multicast.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT dvb.o -MD -MP -MF .deps/dvb.Tpo -c -o dvb.o dvb.c
mv -f .deps/dvb.Tpo .deps/dvb.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT mumudvb.o -MD -MP -MF .deps/mumudvb.Tpo -c -o mumudvb.o mumudvb.c
mv -f .deps/mumudvb.Tpo .deps/mumudvb.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT mumudvb_mon.o -MD -MP -MF .deps/mumudvb_mon.Tpo -c -o mumudvb_mon.o mumudvb_mon.c
mv -f .deps/mumudvb_mon.Tpo .deps/mumudvb_mon.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT mumudvb_common.o -MD -MP -MF .deps/mumudvb_common.Tpo -c -o mumudvb_common.o mumudvb_common.c
In file included from /usr/include/string.h:495,
                 from mumudvb_common.c:40:
In function ‘strncpy’,
    inlined from ‘string_comput’ at mumudvb_common.c:243:2:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:106:10: warning: ‘__builtin_strncpy’ output truncated before terminating nul copying as many bytes from a string as its length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mumudvb_common.c: In function ‘string_comput’:
mumudvb_common.c:234:7: note: length computed here
  234 |   len=strlen(string);
      |       ^~~~~~~~~~~~~~
mv -f .deps/mumudvb_common.Tpo .deps/mumudvb_common.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT network.o -MD -MP -MF .deps/network.Tpo -c -o network.o network.c
mv -f .deps/network.Tpo .deps/network.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rewrite_pmt.o -MD -MP -MF .deps/rewrite_pmt.Tpo -c -o rewrite_pmt.o rewrite_pmt.c
mv -f .deps/rewrite_pmt.Tpo .deps/rewrite_pmt.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rewrite_pat.o -MD -MP -MF .deps/rewrite_pat.Tpo -c -o rewrite_pat.o rewrite_pat.c
mv -f .deps/rewrite_pat.Tpo .deps/rewrite_pat.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rewrite.o -MD -MP -MF .deps/rewrite.Tpo -c -o rewrite.o rewrite.c
mv -f .deps/rewrite.Tpo .deps/rewrite.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rewrite_sdt.o -MD -MP -MF .deps/rewrite_sdt.Tpo -c -o rewrite_sdt.o rewrite_sdt.c
mv -f .deps/rewrite_sdt.Tpo .deps/rewrite_sdt.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rewrite_eit.o -MD -MP -MF .deps/rewrite_eit.Tpo -c -o rewrite_eit.o rewrite_eit.c
mv -f .deps/rewrite_eit.Tpo .deps/rewrite_eit.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT rtp.o -MD -MP -MF .deps/rtp.Tpo -c -o rtp.o rtp.c
mv -f .deps/rtp.Tpo .deps/rtp.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT sap.o -MD -MP -MF .deps/sap.Tpo -c -o sap.o sap.c
mv -f .deps/sap.Tpo .deps/sap.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT ts.o -MD -MP -MF .deps/ts.Tpo -c -o ts.o ts.c
mv -f .deps/ts.Tpo .deps/ts.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT t2mi.o -MD -MP -MF .deps/t2mi.Tpo -c -o t2mi.o t2mi.c
mv -f .deps/t2mi.Tpo .deps/t2mi.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT tune.o -MD -MP -MF .deps/tune.Tpo -c -o tune.o tune.c
mv -f .deps/tune.Tpo .deps/tune.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT unicast_http.o -MD -MP -MF .deps/unicast_http.Tpo -c -o unicast_http.o unicast_http.c
mv -f .deps/unicast_http.Tpo .deps/unicast_http.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT unicast_queue.o -MD -MP -MF .deps/unicast_queue.Tpo -c -o unicast_queue.o unicast_queue.c
mv -f .deps/unicast_queue.Tpo .deps/unicast_queue.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT unicast_EIT.o -MD -MP -MF .deps/unicast_EIT.Tpo -c -o unicast_EIT.o unicast_EIT.c
unicast_EIT.c:28:14: warning: ‘log_module’ defined but not used [-Wunused-variable]
   28 | static char *log_module="Unicast : ";
      |              ^~~~~~~~~~
mv -f .deps/unicast_EIT.Tpo .deps/unicast_EIT.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf_sdt.o -MD -MP -MF .deps/autoconf_sdt.Tpo -c -o autoconf_sdt.o autoconf_sdt.c
mv -f .deps/autoconf_sdt.Tpo .deps/autoconf_sdt.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf_atsc.o -MD -MP -MF .deps/autoconf_atsc.Tpo -c -o autoconf_atsc.o autoconf_atsc.c
mv -f .deps/autoconf_atsc.Tpo .deps/autoconf_atsc.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf_pmt.o -MD -MP -MF .deps/autoconf_pmt.Tpo -c -o autoconf_pmt.o autoconf_pmt.c
mv -f .deps/autoconf_pmt.Tpo .deps/autoconf_pmt.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT autoconf_nit.o -MD -MP -MF .deps/autoconf_nit.Tpo -c -o autoconf_nit.o autoconf_nit.c
mv -f .deps/autoconf_nit.Tpo .deps/autoconf_nit.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT unicast_clients.o -MD -MP -MF .deps/unicast_clients.Tpo -c -o unicast_clients.o unicast_clients.c
mv -f .deps/unicast_clients.Tpo .deps/unicast_clients.Po
gcc -DHAVE_CONFIG_H -I.    -Wall -Wextra -g -O2 -MT unicast_monit.o -MD -MP -MF .deps/unicast_monit.Tpo -c -o unicast_monit.o unicast_monit.c
unicast_monit.c: In function ‘unicast_send_channel_list_xml’:
unicast_monit.c:643:24: error: conflicting types for ‘scam_vars’
  643 |     scam_parameters_t *scam_vars=(scam_parameters_t *)scam_vars_v;
      |                        ^~~~~~~~~
unicast_monit.c:642:11: note: previous declaration of ‘scam_vars’ was here
  642 |     char *scam_vars;
      |           ^~~~~~~~~
make[2]: *** [Makefile:493: unicast_monit.o] Грешка 1
make[2]: излизане от директория „/opt/MuMuDVB/src“
make[1]: *** [Makefile:346: all] Грешка 2
make[1]: излизане от директория „/opt/MuMuDVB/src“
make: *** [Makefile:347: all-recursive] Грешка 1

```

**Comment by anphsw** [comment]
Seems like actual error.

**Comment by anphsw** [comment]
Fixed by #297
Please try to compile.
